.github/workflows/update-openssl.yml

```diff
@@ -21,11 +21,13 @@ jobs:
         run: |
           ./tools/dep_updaters/update-openssl.sh download > temp-output
           cat temp-output
-          tail -n1 temp-output | grep "NEW_VERSION=" >> "$GITHUB_ENV"
+          tail -n1 temp-output | grep "NEW_VERSION=" >> "$GITHUB_ENV" || true
+          tail -n2 temp-output | grep "HAS_UPDATE=" >> "$GITHUB_ENV" || true
           rm temp-output
         env:
           GITHUB_TOKEN: ${{ secrets.GH_USER_TOKEN }}
       - name: Create PR with first commit
+        if: env.HAS_UPDATE
```

Comment:
Why define a different variable rather than using `NEW_VERSION`?

Reply:
yes, now it should be good